app/mailers/visitor_mailer.rb

```diff
@@ -13,7 +13,7 @@ class VisitorMailer < ActionMailer::Base
   attr_reader :visit
   helper_method :visit
 
-  default('List-Unsubscribe' => '<https://www.prisonvisits.service.gov.uk/unsubscribe>')
+  default('List-Unsubscribe' => Rails.configuration.unsubscribe_url)
```

Comment:
The config value includes the `<` `>`.

They do appear to be necessary - well according to the examples anyway. (http://www.faqs.org/rfcs/rfc2369.html)